### PR TITLE
#38513

### PR DESCRIPTION
**Pull request by Carbonhell**
Re-mirrors https://github.com/tgstation/tgstation/pull/38513

### CODE REVIEW

**Review by Jujumatic — APPROVED** [review]
looks gud bboiiii

File: code/controllers/subsystem/throwing.dm [hunk 1]
```diff
@@ -113,15 +122,15 @@ SUBSYSTEM_DEF(throwing)
 			return
 
 /datum/thrownthing/proc/finalize(hit = FALSE, target=null)
-	set waitfor = 0
-	SSthrowing.processing -= thrownthing
+	set waitfor = FALSE
```
Comment (Jujumatic) [inline code/controllers/subsystem/throwing.dm:125]: why tho

Comment (Carbonhell) [reply code/controllers/subsystem/throwing.dm:125]: go sleep ur drunk

Comment (Jujumatic) [reply code/controllers/subsystem/throwing.dm:125]: no u !!!!!11

**Comment by GuyonBroadway** [comment]
this seems to be identical to the tg one (despite some lines not being marked as added but they are there, i checked) so I'm sure this si safe, it it fucks up, blame tg.

**Pull request merged**